.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk: 
+  - openjdk8
```

Comment:
I suppose, only 8 supported great by kotlin

Reply:
Actually java6-12 are supported by Kotlin (Just that kolin does not use any features of java9+ and does nt compile to those bytecode levels, yet) so I think it is worth testing this support for this library on CI.